common/models/picks-user.js

```diff
@@ -94,12 +140,7 @@ module.exports = function(PicksUser) {
       process.env.FACEBOOK_APP_SECRET_PRODUCTION :
       process.env.FACEBOOK_APP_SECRET_STAGING;
       var Facebook = PicksUser.app.dataSources.Facebook;
-      var Group = PicksUser.app.models.Group;
-      var Score = PicksUser.app.models.Score;
-      var Season = PicksUser.app.models.Season;
-      var isNewUser = false;
-      var globalGroup;
-      let userResponse, accessTokenId, newUser, newScore, facebookAccessToken;
+      let userResponse, facebookAccessToken, facebookId;
       // Exchange oauth token for access token
       Facebook.accessToken(appId, appSecret, redirectUrl, oauthToken)
       .then(function(response) {
```

Comment:
Remove this since it's not used

Reply:
Which part exactly?

Reply:
line 146; the whole `then` chain

Reply:
No its used

Reply:
The accessToken method returns a promise thats handled by the then chain.

Reply:
You're right, I didn't see the collapsed snippet